### ISSUE TITLE
Test full build on new set-up

### ISSUE THREAD

**Issue by atumscott** [nptscot/npt#150]

**Comment by Robinlovelace** [comment]
Re-opening as has not been fixed, after #151 was merged prematurely.

Issue I'm seeing: 

```
4537: deleted
Saving tmp/batch_ebike_school_7_with_6944_routes_of_126944_rows.Rds to tmp
Saving tmp/od_school_7_with_6944_of_126944_rows.Rds to tmp
Combining results
Saving outputdata/2023-06-06/routes_max_dist_school_ebike.Rds
• built target r_school
• start target calculate_benefits
• built target calculate_benefits
• start target zone_stats
• built target zone_stats
• start target uptake_list_commute
Uptake for fastest
Uptake for balanced
Uptake for quietest
✖ Problem with the pipeline.
Error:
! problem with the pipeline.
```

**Comment by Robinlovelace** [comment]
This is now done, resulting data from full build: https://github.com/nptscot/outputdata/releases
